### PR TITLE
added link around badge to travisci

### PR DESCRIPTION
**Pull request by lkwg82**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,118 @@
-![travisci](https://travis-ci.org/lewisd32/lint-maven-plugin.svg)
+[![travisci](https://travis-ci.org/lewisd32/lint-maven-plugin.svg)](https://travis-ci.org/lewisd32/lint-maven-plugin)
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.lewisd/lint-maven-plugin/badge.svg)](https://search.maven.org/#search%7Cga%7C1%7Cg%3Acom.lewisd)
+
+
+# de.lgohlke.selenium-webdriver
```
Comment (lewisd32) [inline README.md]: I don't think you meant to add all this. :)

**Comment by lewisd32** [comment]
I'm not sure about why this build is failing on Travis CI.  It passes on my Jenkins server.  The failing build says to see "/home/travis/build/lewisd32/lint-maven-plugin/target/it/pass-issue-23-onlyRunRules-ignored/build.log", but I'm not sure if I can access that on Travis CI.

File: README.md (hunk 1 above)
Comment (lkwg82) [inline README.md]: ooops, of course not *blush*

**Comment by lkwg82** [comment]
this is because of something different, because it only changes the README.md

**Comment by lewisd32** [comment]
Yeah, the failure is because of the integration test added in https://github.com/lewisd32/lint-maven-plugin/pull/28, but I don't know why it's failing.

**Comment by lkwg82** [comment]
It fails because of referencing 0.0.10-SNAPSHOT instead of new 0.0.11-SNAPSHOT. Thats all.

**Comment by lewisd32** [comment]
Oh, right, that makes sense.  I'll try to fix that today.  I'd like to get master green on TravisCI before I merge any more branches.

**Comment by lewisd32** [comment]
Ok, I was able to get master happy on TravisCI.  I think your change to the integration test pom is causing a conflict now.  Once the README.md is cleaned up (from the accidental de.lgohlke.selenium-webdriver stuff) I can merge this.

**Pull request merged**